src/HttpApplicationRunner.php

```diff
@@ -5,6 +5,7 @@
 namespace Yiisoft\Yii\Runner\Http;
 
 use ErrorException;
+use HttpSoft\Message\ResponseFactory;
```

Comment:
I don't think that the package should be linked with that package directly

Reply:
Done